curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-algorithmic-thinking-by-building-a-dice-game/657e0c2c6a9d37705146f34d.md

```diff
@@ -21,6 +21,12 @@ You should have a `detectFullHouse` function.
 assert.isFunction(detectFullHouse);
 ```
 
+Your `detectFullHouse` function should have a parameter.
+
+```js
+assert.match(code, /((const\s+detectFullHouse\s*=\s*(\(\s*\w+\s*\))\s*=>\s*)|(function\s*detectFullHouse\(\s*\w+\s*\)))/);
```

Comment:
The arrow function should allow `var` and `let`, and the parentheses should be optional when it is declared with just a single parameter.

The space between the `function` keyword and the name is not optional.

Something like this should work.

```suggestion
assert.match(code, /(?:var|let|const)\s+detectFullHouse\s*=\s*(\(\s*\w+\s*\)|\s*\w+\s*)\s*=>\s*|function\s+detectFullHouse\s*\(\s*\w+\s*\)/);
```